cuqipy_fenics/utilities.py

```diff
@@ -1,6 +1,62 @@
+import numpy as np
 import dolfin as dl
+import warnings
 from numbers import Number
 
+
+def compute_stats(samples):
```

Comment:
1) Since this is specific to cuqi samples object I would suggest adding a type hint here
2) Originally I thought this method would be used to compute general stats such as mean, var etc. using the FEniCS objects by the geometries. At the moment it looks like a utility method used by the user, but perhaps there is a way to incorporate it such that Geometry works with it? I would suggest creating an issue for this in this repo.

Reply:
Good point! I added the hint as well as an issue #60 

The second approach that compute_stats follows, gives the same result as what is obtained by the command `samples.funvals.vector.variance(). I also made it hidden (_compute_stats) since I mainly use it in tests and one demo for verification, but I do not think we should confuse the user by this method since we expect the user would use `samples.funvals.vector.variance()`

Reply:
Great thanks!